cargo-concordium/src/main.rs

```diff
@@ -313,11 +313,16 @@ struct VerifyOptions {
     #[structopt(
         name = "source",
         long = "source",
-        help = "Path to the sources. If not present then the sources will be downloaded from an \
+        help = "Path to the source code of the code, this would be the tar produced during the \
+                reproducible build. If not present then the sources will be downloaded from an \
                 embedded link in the build info."
     )]
     source_path:       Option<PathBuf>,
-    #[structopt(name = "module", long = "module", help = "Module to verify.")]
+    #[structopt(
+        name = "module",
+        long = "module",
+        help = "Path to the module WASM file to verify."
```

Comment:
Concordium smart contract modules are actually not valid Wasm modules... we prefix a few bytes, making them invalid, so maybe:
```suggestion
        help = "Path to the smart contract module file to verify."
```